web3/beacon/async_beacon.py

```diff
@@ -64,7 +67,7 @@ def __init__(
         request_timeout: float = 10.0,
     ) -> None:
         self.base_url = base_url
-        self.request_timeout = request_timeout
+        self.request_timeout = ClientTimeout(request_timeout)
```

Comment:
Yeah we definitely need this somewhere. I wonder if it's best to save the value as a `float` still so if a user decides to change this on the instance via `beacon.request_timeout = ...` then it's still configured as a float?

Whichever way we go I think this needs to happen. If we go the latter route we can wrap the value in a `ClientTimeout` when we make the call within `_async_make_get_request()`, setting `timeout=ClientTimeout(self.request_timeout)`.

Thoughts there?

Reply:
Yes I agree, please have a look at the updated version I've pushed.